app/View/Components/UserCard.php

```diff
@@ -15,16 +13,29 @@ class UserCard extends Component
      *
      * Date of the users first chirp || null if user has no chirps
      */
+    public $name;
+    public $profileImageUrl;
```

Comment:
you can refactor this into a settings object or array so that you don't have to list all of the attributes here, but this is great too!